apps/02RockPaperScissors.js

```diff
@@ -7,8 +7,41 @@ prompt.start();
 
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
+  if (hand1 === hand2) {
```

Comment:
Good job. We'll talk about how to shrink the code down in next session. Good job on indentation again!